detection_operator_runner.py

```diff
@@ -26,6 +26,7 @@
 import pylot.flags
```

Comment:
It might be a good idea to rename the file as it's not a detection_operator_runner anymore.

Reply:
Agreed. I'll keep it for now since changing the name will cause problems with other open PRs. @objorkman can you change the name once your open PRs are merged?